pkg/hooks/exec/exec_test.go

```diff
@@ -29,7 +29,11 @@ func TestRun(t *testing.T) {
 		Args: []string{"sh", "-c", "cat"},
 	}
 	var stderr, stdout bytes.Buffer
-	hookErr, err := Run(ctx, hook, []byte("{}"), &stdout, &stderr, DefaultPostKillTimeout)
+	cwd, err := os.Getwd()
```

Comment:
Why not just do this inside of the Run function, rather then copying the same code everywhere?  Do you invision calling Run with anything other the os.Getwd()?

Reply:
To clarify, the passed in cwd value suppose to be something like

```
/home/user/.local/share/containers/storage/overlay-containers/7972768fe111b9d9dfab88b4e2dd0b8bf616f66e9c461a6c8506d5dc2986738a/userdata
```

i.e, the bundle directory of the container.

For now it's passing the current cwd value of podman process, be it a daemon or standalone command line process. So it could be something like `/home/myuser/`.

The only reason I pass `os.Getwd` value in the tests is because those tests don't care about the cwd value. Only the new one I added for testing cwd cares about the cwd argument. Hope that makes sense.

But I understand that's probably not a very smart way to write test this way I guess, any suggestion to DRY this part? I guess I should be able to pass in empty string or something like that to make it as the "empty" value so that the current cwd value of process will be used as well? I haven't write go for a while so I guess it might need to give it a try tomorrow.